README.md

```diff
@@ -111,57 +97,68 @@ This transferred between frontend and backend by cookies within the http request
 All interactions on entities which are exposed for http requests are protected the WebSecurity configuration
 which only allows to call the /players endpoint without any valid authentication.
 
-| File                                                                                                                                                                                                  | Responsability                                                                                                    |
-|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
-| [PlayerEventHandler](https://github.com/sopra-fs22-group-36/screw-your-neighbor-server/blob/main/src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/sideeffects/PlayerEventHandler.java) | Creates security context with authentication token when a new player is created.                                  |
```

Comment:
You can (and should) use relative URL's in the project README.
Then it links to the file in the branch you opened on github.
And the link also works in your IDE.
And the link is shorter